ethereum.star

```diff
@@ -1,70 +1,92 @@
+            "eth1_follow_distance": 2048,
+            # MIN_VALIDATOR_WITHDRAWABILITY_DELAY, Default: 256
+            # A validator can stop participating once it has made it through the exit queue. However, its stake remains locked for the duration of MIN_VALIDATOR_WITHDRAWABILITY_DELAY. This is to allow some time for any slashable behaviour to be detected and reported so that the validator can still be penalised (in which case the validator's withdrawable time is pushed EPOCHS_PER_SLASHINGS_VECTOR into the future).
+            # Once the MIN_VALIDATOR_WITHDRAWABILITY_DELAY period has passed, the validator becomes eligible for a full withdrawal of its stake and rewards on the next withdrawals sweep, as long as it has ETH1_ADDRESS_WITHDRAWAL_PREFIX (0x01) withdrawal credentials set. In any case, being in a "withdrawable" state means that a validator has now fully exited from the protocol.
+            "min_validator_withdrawability_delay": 256,
+            # SHARD_COMMITTEE_PERIOD, Default: 256
+            # This really anticipates the implementation of data shards, which is no longer planned, at least in its originally envisaged form. The idea is that it's bad for the stability of longer-lived committees if validators can appear and disappear very rapidly.
+            # Therefore, a validator cannot initiate a voluntary exit until SHARD_COMMITTEE_PERIOD epochs after it has been activated. However, it could still be ejected by slashing before this time.
+            "shard_committee_period": 256,
+            # GENESIS_DELAY, Default: 12
+            # This is a grace period to allow nodes and node operators time to prepare for the genesis event. The genesis event cannot occur before MIN_GENESIS_TIME. If MIN_GENESIS_ACTIVE_VALIDATOR_COUNT validators are not registered sufficiently in advance of MIN_GENESIS_TIME, then Genesis will occur GENESIS_DELAY seconds after enough validators have been registered.
+            "genesis_delay": 12,
         },
-    )
+        "additional_services": args["l1_additional_services"],
+        "port_publisher": port_publisher,
+    }
+
+    # Enable Pectra hardfork if needed.
+    if args.get("pectra_enabled", False):
+        # Note: The electra fork epoch is set to 1 instead of 0 to avoid the following error in the CL node (lighthouse).
+        #  Mar 11 11:56:46.595 CRIT Failed to start beacon node             reason: Built-in genesis state SSZ bytes are invalid: OffsetOutOfBounds(522733568)
+        l1_args["network_params"]["electra_fork_epoch"] = 1
+        # Note: The fulu fork epoch is set to a multiple of 256 to avoid the following warning in the CL node (lighthouse).
+        # Mar 11 11:44:39.231 WARN Fork boundaries are not well aligned / multiples of 256, misaligned_forks: [(Fulu, Epoch(100000001))], info: This may cause issues as fork boundaries do not align with the start of sync committee period.
+        l1_args["network_params"]["fulu_fork_epoch"] = 256
```

Comment:
Should this be a bigger multiple?? I don't know if you want to randomly trigger this in 256 seconds (i think in some case we use 1 second epochs).

Reply:
I just removed it since it's a different hardfork: [test: remove fulu fork epoch](https://github.com/0xPolygon/kurtosis-cdk/pull/524/commits/d0769ae4359cecd1b5ca7e74867496ad277bf28c)